WELD.md

```diff
@@ -139,3 +104,36 @@ python weld.py
 ### Goal:
 
 1.  Track the submodule (recipe) version with the current data repository.
+
+## Step 5: Commit most weld results to Github
+
+### Procedure:
```

Comment:
```suggestion
### Procedure:

```

Add space after H3 heading (nitpicky style suggestion!)

(Note that you can simply "accept suggestion" and this commit will engage to change the actual file. Not sure if you've seen this before, many people haven't!)